src/main/java/com/thebois/models/beings/Pawn.java

```diff
@@ -1,20 +1,20 @@
 package com.thebois.models.beings;
 
+import java.util.Random;
+
 import com.thebois.models.Position;
 
 /**
  * Pawn.
  */
 public class Pawn extends AbstractBeing {
 
-    /**
-     * Creates a Pawn.
-     */
-    public Pawn() {
-    }
+    /* Temporary hard-coded world size. Should be removed when pathfinding is implemented. */
+    private static final int WORLD_SIZE = 50;
```

Comment:
Can world size be a static in `World` that we assign in its constructor? (Like if its possible in JAVA) If so we can remove this field and have WORLD.size

What do you think?

Reply:
Feels a bit random that the pawn has a hardcoded value of the world, since we can change the value at  any time, and then we have to update it here aswell

Reply:
"/* Temporary hard-coded world size. Should be removed when pathfinding is implemented. */" 

It was either that, or send a reference for the `World` instance to `Pawn` which is even worse. 

That's a nice idea, we could place it as a `public static` constant in `World`. Problem with that is that is global data, which is still a huge code smell. 

My reasoning was that no matter how we implement this right now, it's too much work for something that'll be removed right after this PR is merged and I begin work on #13 😛

Reply:
> 
> 
> "/* Temporary hard-coded world size. Should be removed when pathfinding is implemented. */"

Aaaaaaaaaaaaah, I see. Then it's sort of OK, it's pretty much a `TODO: ` comment.

> 
> It was either that, or send a reference to the `World` instance to `Pawn` which is even worse.

That I agree 100% with! 👍 

> 
> That's a nice idea, we could place it as a `public static` constant in `World`. Problem with that is that is global data, which is still a huge code smell.

Hmm, true.

> 
> My reasoning was that no matter how we implement this right now, it's too much work for something that'll be removed right after this PR is merged and I begin work on #13 😛

As long as we don't forget it, so it because a bug in the future 😄

Reply:
>it's pretty much a TODO: comment.

Yeah... Which hurts to merge into master, but it's either that or have ugly dependencies, or just completely remove it and have pawns that don't move.

>As long as we don't forget it, so it because a bug in the future 😄

![image](https://user-images.githubusercontent.com/12572888/134356107-79f1cec7-4d46-4fcb-8688-99df194bc536.png)